### PR TITLE
update(faq/search): add explanation for suitable torrent count differences

### PR DESCRIPTION
**Pull request by convexshiba**
add entry for how to debug why torrents are not "suitable for search"

### CODE REVIEW

File: docs/basics/faq-troubleshooting.md [hunk 1]
```diff
@@ -424,3 +424,8 @@ can match from multiple sources. `cross-seed` is simply offering you the ability
 to revive the stalled torrents inside your client.
 
 :::
+
+### Why do I have so many torrents that are not being searched, a.k.a not "suitable to search for matches"? 
+
+Most likely this is expected behavior due to `config.js` settings such as `includeSingleEpisodes`,
+`includeNonVideos`. Check verbose log in app config folder for `[prefilter]` entries.
```
Comment (zakkarry) [inline docs/basics/faq-troubleshooting.md:431]:
```suggestion
### Why are some torrents not suitable for searching?

Seeing a difference in total torrent count and suitable torrents (as well as
unique searches) is expected behavior. This is usually due to `config.js`
settings such as `includeSingleEpisodes`or `includeNonVideos`, `exclude` options
based on age or last date of search, or duplicate search queries which have been
reduced down to a single query.

:::tip

You can check the verbose log in the log directory of the app config folder for
`[prefilter]` entries immediately before a search starts for exact details.

:::
```

I know this is a pretty dramatic elaboration/rework, but let me know what you think and we can work from here. 

Also, not entirely sure placing this at the very end is where I'd put it, but we can worry about the placement later (I try and keep super common things more near the top) - and this seems more common than the very end. But I understand why you placed it there lol.

**Review by zakkarry — CHANGES_REQUESTED** [review]

**Review by convexshiba — COMMENTED** [review]
adopt suggestions

**Comment by zakkarry** [comment]
> adopt suggestions

Any opinion on placement more near the top?

**Comment by convexshiba** [comment]
Nah not at all.

**Review by convexshiba — COMMENTED** [review]
order

**Comment by zakkarry** [comment]
thanks

**Review by zakkarry — APPROVED** [review]

**Pull request merged**